modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro

```diff
@@ -1,5 +1,54 @@
 definition {
 
+	macro addNotificationTemplateViaAPI {
```

Comment:
`addEmailNotificationTemplateViaAPI`

We should be clearer here what type of notification template we're creating